ITEM.tsv

```diff
@@ -5726,7 +5726,7 @@ ITEM_20150717_005725	Someone told this weapon is directly related to Ruclyus, bu
 ITEM_20150717_005726	Someone told considering the disaster encountered Roxona, It would be great if these weapons have an immunity against the petrifation. Although sorry for Roxona of petrified city, it's surely that this weapon is better than the other side.
 ITEM_20150717_005727	People may not know about it but the reason why they can get this for a cheap price is because of the plans and foresight of Goddess Laima. 
 ITEM_20150717_005728	No one knows how Jukopus give this weapon its ablities. Winterspoon family seems to know but do not share it. 
-ITEM_20150717_005729	The demon lord Mirtis said that before he used the Bubbes, Bubbes carried no equipment with magical power in them.
+ITEM_20150717_005729	The demon lord Mirtis said that before he used the Vubbes, Vubbes carried no equipment with magical power in them.
```

Comment:
You have it as "The demon lord Mirtis said that before **he** used Vubbes", or am I missing some context here? Also, I just saw that in #1059 Demon Lord was capitalized.